js/app.js

```diff
@@ -25,120 +25,121 @@ $(function () {
     .addView(printView);
 
   // Instantiate the controllers
-  var welcomeViewController =
+  const welcomeViewController =
     new WelcomeViewController(welcomeView, model, generalController);
-  var sideBarViewController =
+  const sideBarViewController =
     new SideBarViewController(sideBarView, model, generalController);
-  var dishSearchViewController =
+  const dishSearchViewController =
     new DishSearchViewController(dishSearchView, model, generalController);
-  var dishDetailsViewController =
+  const dishDetailsViewController =
     new DishDetailsViewController(dishDetailView, model, generalController);
-  var dinnerOverviewViewController =
+  const dinnerOverviewViewController =
     new DinnerOverviewViewController(dinnerOverviewView, model, generalController);
-  var titleBarViewController =
+  const titleBarViewController =
     new TitleBarViewController(titleBarView, model, generalController);
-  var printViewController =
+  const printViewController =
     new PrintViewController(printView, model, generalController);
-  var dishItemViewController =
+  const dishItemViewController =
     new DishItemViewController($("#searchedDishes"), model, generalController);
 
-  function getCookie(cname) {
-    var name = cname + "=";
-    var ca = document.cookie.split(';');
-    for (var i = 0; i < ca.length; i++) {
-      var c = ca[i].trim();
-      if (c.indexOf(name) == 0) return c.substring(name.length, c.length);
-    }
-    return "";
-  }
-
-  var SCREENS = [
+  const SCREENS = [
     "WELCOME",
     "SELECT_DISH",
     "DISH_DETAILS",
     "DINNER_OVERVIEW",
     "DINNER_PRINTOUT"
   ];
-  // this set is created from the SCREENS list
-  // key and value are the same
-  SCREENS_Set = new Set(SCREENS); // this should be global
```

Comment:
Good point about SCREENS_Set, I was about to simplify the procedure to write screen names each time, but it turns out to be the same with writing string of names manually. It could be better if we store these STATIC strings in a global context, for instance, a dedicated .js file. Maybe we could do that later